framework/test/unit/modules/token/commands/transfer.spec.ts

```diff
@@ -160,277 +201,268 @@ describe.skip('Transfer command', () => {
-
-		beforeEach(async () => {
-			stateStore = new PrefixedStateReadWriter(new InMemoryPrefixedStateDB());
-			const userStore = tokenModule.stores.get(UserStore);
-			const context = createTransientMethodContext({ stateStore });
-			await userStore.save(
-				context,
-				address.getAddressFromPublicKey(sender.publicKey),
-				localTokenID,
-				{ availableBalance: senderBalance, lockedBalances: [] },
-			);
-			await userStore.save(
-				context,
-				address.getAddressFromPublicKey(sender.publicKey),
-				secondTokenID,
-				{ availableBalance: senderBalance, lockedBalances: [] },
-			);
-			await userStore.save(
-				context,
-				address.getAddressFromPublicKey(sender.publicKey),
-				thirdTokenID,
-				{ availableBalance: senderBalance, lockedBalances: [] },
-			);
-			await userStore.save(
-				context,
-				address.getAddressFromPublicKey(recipient.publicKey),
-				localTokenID,
-				{ availableBalance: recipientBalance, lockedBalances: [] },
-			);
-			const supplyStore = tokenModule.stores.get(SupplyStore);
-			await supplyStore.set(context, localTokenID.slice(4), { totalSupply });
-		});
-
-		it.skip('should reject when sender does not have enough balance for amount', async () => {
+		it('should fail for non existent account when initialization fee is not what is configured in init', async () => {
 			const context = createTransactionContext({
-				stateStore,
 				transaction: new Transaction({
 					module: 'token',
 					command: 'transfer',
-					fee: BigInt(0),
+					fee: BigInt(5000000),
 					nonce: BigInt(0),
-					senderPublicKey: sender.publicKey,
+					senderPublicKey: utils.getRandomBytes(32),
 					params: codec.encode(transferParamsSchema, {
-						tokenID,
-						amount: senderBalance + BigInt(1),
+						tokenID: Buffer.from('0000000100000000', 'hex'),
+						amount: BigInt(100000000),
 						recipientAddress: utils.getRandomBytes(20),
 						data: '1'.repeat(64),
+						accountInitializationFee: BigInt(USER_SUBSTORE_INITIALIZATION_FEE) - BigInt(1),
 					}),
 					signatures: [utils.getRandomBytes(64)],
 				}),
 			});
-			await expect(
-				command.execute(context.createCommandExecuteContext(transferParamsSchema)),
-			).rejects.toThrow('balance 200000000 is not sufficient');
+			const result = await command.verify(context.createCommandVerifyContext(transferParamsSchema));
+			expect(result.status).toEqual(VerifyStatus.FAIL);
+			expect(result.error?.message).toInclude('Invalid account initialization fee.');
 		});
 
-		it('should resolve when recipient exist for different tokenID but does not have enough balance for minBalance', async () => {
+		it('should fail if token balance for tokenID TOKEN_ID_STORE_INITIALIZATION is less than the sum of configured initialzation fee and transaction amount', async () => {
+			const amount = BigInt(100000000);
+
+			command['_method'].getAvailableBalance = jest
+				.fn()
+				.mockResolvedValue(amount + BigInt(USER_SUBSTORE_INITIALIZATION_FEE) - BigInt(1));
```

Comment:
```suggestion
			jest
				.spyOn(command['_method'], 'getAvailableBalance')
				.mockResolvedValue(amount + BigInt(USER_SUBSTORE_INITIALIZATION_FEE) - BigInt(1));
```
same for other cases